.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,28 @@
+# dependabot[bot]
+
+name: Dependabot Auto Merge
+
+on:
+  check_suite:
+    types:
+      - completed
+  pull_request:
+    types:
+      - edited
+      - labeled
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+      - unlabeled
+      - unlocked
+
+jobs:
+  worker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: ridedott/merge-me-action@master
```

Comment:
Please use a specific version tag.

Reply:
Yeah, I would also enable GitHub Actions autoupdate in dependabot

Reply:
I wouldn't go that far. It doesn't honor beta stuff. Like right now, `setup-node@v1` is current but that bot will suggest updating to `setup-node@v2` which is in beta.

Reply:
I don't see anything bad in this beta, nvm, I'm ok to go with current solution